evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -291,25 +291,9 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
         }
 
         function repeat(action) {
-            let answer = slider.find("[data-layer=2]");
-
-            if(action === "start" && answer.length === 0) {
-                slideLayer(2, "right", slider.find(".alert"));
-                return;
-            }
-
-            let reviewed = [];
-            let unreviewed = [];
-
-            answer.each(function() {
-                let item = $(this);
-                if(item.is("[data-review]")) {
-                    reviewed.push(item);
-                } else {
-                    unreviewed.push(item);
-                }
-            });
-
+            let answers = slider.find("[data-layer=2]");
+            let reviewed = answers.filter("[data-review]").toArray().map($);
+            let unreviewed = answers.filter(":not([data-review])").toArray().map($);
```

Comment:
`toArray()` will put a jQuery collection (an array-like object) into a plain array, but as DOM objects. A map on `$` will convert them back into jQuery objects. Nothing else has been done in the loop, where the first line was `let item = $(this)`

Reply:
but why do you need it at all? to me your answer reads as "...converts an array(-like object) with jquery objects to an array with jquery objects".

Reply:
Aha, you think that his is so easy? The array-like object can be indexed with `.get(i)` not `[i]`, which is not a great harm, but the problem lies in the combination of `unreviewed` and `reviewed` sets. Yes, sets is a good keyword: the ordering will reset to markup ordering. I could move the elements in the DOM, but this would result in a more complex markup (every item must know the name of their parent and the parent must know their name and the django template loop must be run three times otherwise the ordering of the layers would be wrong and then then we need CSS flex `order` to bring back peace into the anarchy), as the parent (or reference) is quite simply found in the markup by going backwards. Yes, this is one piece which I glued together from jQuery docs, an outdated (?) answer from SO and modern JS and I don't think it is ideal, but it guarantees an array with the ordering we want which makes less headache elsewhere.

Reply:
would https://api.jquery.com/jQuery.merge/ be of any help?